qiita_pet/handlers/study_handlers/listing_handlers.py

```diff
@@ -327,11 +203,29 @@ def get(self, ignore):
             study_proc = proc_samples = None
         info = _build_study_info(self.current_user, study_proc=study_proc,
                                  proc_samples=proc_samples)
+        # linkifying data
```

Comment:
Can all this be done at the end of _build_study_info instead that in the handler? I think it will be easier to debug and even to offload at some point if we want to.